src/App.tsx

```diff
@@ -7,14 +7,32 @@ import postsFromServer from './api/posts';
 import commentsFromServer from './api/comments';
 import usersFromServer from './api/users';
 
+import { User, Comment } from './types';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  if (foundUser) {
+    return foundUser;
+  }
+
+  return null;
+}
```

Comment:
You can condense these lines into a single line that returns a value

For ex
`usersFromServer.find(user => user.id === userId) || null`